dao/src/main/java/com/epam/esm/dao/sort/GiftCertificateSorter.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.esm.dao.sort;
+
+import com.epam.esm.dao.entity.GiftCertificate;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Order;
+import javax.persistence.criteria.Root;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GiftCertificateSorter {
+
+  public void sort(
+      CriteriaQuery<GiftCertificate> criteria,
+      CriteriaBuilder builder,
+      Root<GiftCertificate> root,
+      List<String> sorts) {
+    List<Order> orders = new ArrayList<>();
+    nameSorting(builder, root, sorts, orders);
+    descriptionSorting(builder, root, sorts, orders);
+    createDateSorting(builder, root, sorts, orders);
+    lastUpdateDateSorting(builder, root, sorts, orders);
+    criteria.orderBy(orders);
+  }
+
+  private void lastUpdateDateSorting(
```

Comment:
This is method, it is not sorting, but sort

Reply:
Thank you for noticing my mistake. (fixed)